libs/math/include/math/vec4.h

```diff
@@ -84,10 +84,16 @@ class MATH_EMPTY_BASES TVec4 :
 
     // default constructor
     constexpr TVec4() = default;
+#if 0
```

Comment:
remove`#if 0`

Reply:
Can we remove the #if 0 block?

Reply:
wait, if you remove the *whole* `#if 0 ... #endif` block, you have errors?  That's not possible, this is just pre-processor stuff.